modules/ding_ipe_filter/css/ipe.css

```diff
@@ -0,0 +1,53 @@
+/**
+ * @file
+ * Styles for the menu block.
+ */
+
+/**
+ * Override styles for the IPE blocks,
+ * in order to display properly the filter menu.
+ */
+div.panels-ipe-newblock {
+  z-index: 0;
+}
+div.panels-ipe-newblock a.active {
+  z-index: 1;
+}
+div.panels-ipe-placeholder {
+  position: relative;
+}
+div.ipe-popup {
+  -webkit-box-shadow: none;
+  box-shadow: none;
+  position: absolute;
+  z-index: 100;
+  top: 78px;
+  overflow-y: scroll;
+  width: 400px;
+  height: 200px;
+}
+div.ipe-popup ul {
+  padding-left: 0;
+  width: 100%;
+}
+.hide {
+  display: none;
+}
+.left {
+  left: 245px;
+}
+.top {
+  top: -157px !important;
+}
```

Comment:
This targets some very board selectors with which does not seem to be directly related to IPE: `.hide`, `.left`, `.top`, `.layout-wrapper .group-blocks--wrapper`.

I would suggest replacing `.hide` with `.element-hidden` which is a standard part of Drupal which does the same thing.

`.left` and `.top` has styling that seems very specific to our use case. How about renaming them to something which communicates the scope more clearly e.g. `ding-ipe-popup-left`?